src/Frame/Frame.cc

```diff
@@ -2495,8 +2424,8 @@ bool Frame::GetDownsampledRasterData(
         int y_max = bounds.y_max() - 1;
 
         auto tile_stokes_section = GetImageSlicer(AxisRange(x_min, x_max), AxisRange(y_min, y_max), AxisRange(z), stokes);
-        tile_data.resize(tile_stokes_section.slicer.length().product());
-        if (!GetSlicerData(tile_stokes_section, tile_data.data())) {
+        tile_data.resize(tile_stokes_section.length().product());
+        if (!GetSlicerData(tile_stokes_section, stokes, tile_data.data())) {
```

Comment:
Should we consistently use `stokes_index` for int variable rather than `stokes`? (guilty of this myself)